components/fxa-client/src/lib.rs

```diff
@@ -40,6 +40,11 @@ mod account;
 mod auth;
 mod device;
 mod error;
+// We need to make the `internal` mod public for `sync-test` to work
```

Comment:
I think we fixed that so it's no longer true. I put a PR up for your branch to kill this!

Reply:
Sounds great to me.  I just merged the branch in.